exercises/practice/robot-simulator/src/test/groovy/RobotSimulatorSpec.groovy

```diff
@@ -0,0 +1,80 @@
+import spock.lang.*
+
+class RobotSimulatorSpec extends Specification {
+    def "Create robot at origin facing north"() {
+        expect:
+        RobotSimulator results = new RobotSimulator(x, y, direction)
+        RobotSimulator expected = new RobotSimulator(x, y, direction)
+        results.properties == expected.properties
+
+        where:
+        label                               || x    || y   || direction
+        "at origin facing north"            || 0    || 0   || "north"
+        "at negative position facing south" || -1   || -1   || "south"
+    }
+
+
+    @Unroll
```

Comment:
This also applies to the subsequent tests.

```suggestion
    @Ignore
    @Unroll
```

Reply:
Thanks. I wanted to add it but VSCode seemed to indicate the ignore was invalid. I’ll take another look.

Reply:
The other exercises have it, so 🤷

Reply:
Should be fixed now.